### DISCUSSION TITLE
Issue of only showing a white widget on ios 18 when you set the icons on homepage to be white/black and white

### DISCUSSION THREAD

**Discussion by tarekeleter** [community/community#145032] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.185.1 (147102269)
Device: iPhone 12 Pro Max (iOS 18.1)
</details>

**Comment by stevepopovich** [comment]
Hi @teleter I am bit confused about the issue here, do you have a screenshot?

**Reply by tarekeleter** [reply]
This is the normal way the github oages widgets are displayed![image](https://github.com/user-attachments/assets/43fd69ea-cc11-4b25-a421-4bf9ead16807)

**Reply by tarekeleter** [reply]
This is the view of the github widget on ios when you set the icons to be tinted white on ios 18 ![image](https://github.com/user-attachments/assets/8e88cbcc-9430-4bb8-a959-223149785bdc)

**Reply by tarekeleter** [reply]
@stevepopovich you can reproduce this by long press on ios 18 homepage > edit button top left > customise > tinted icon > set to white

**Reply by stevepopovich** [reply]
oh great, thank you very much!

**Reply by stevepopovich** [reply]
Let me look into this.

**Reply by stevepopovich** [reply, marked as answer]
Hi @teleter I have reported this issue, we will be looking into fixing this soon. Thanks for reporting!

**Comment by tarekeleter** [comment]
> Hi @teleter I am bit confused about the issue here, do you have a screenshot?

Seems like this has been fixed in the latest update ![image](https://github.com/user-attachments/assets/1bfbb443-e486-4f54-9388-ce19dee5ac0e)
